model/task_lifecycle.go

```diff
@@ -761,6 +761,16 @@ func UpdateBuildAndVersionStatusForTask(taskId string, updates *StatusChanges) e
 		return errors.WithStack(err)
 	}
 
+	githubCheckTasks := map[string]bool{}
+	githubCheckStatus := ""
```

Comment:
I don't know if there are any realistic scenarios where this can happen, but from the way githubCheckStatus is set we could potentially log a blank status. How do you feel about initially setting it to either passing or failed, and changing it based on the logic below?

Reply:
Well, we only log if `githubCheckStatus != ""` (line 817), so I don't think it will be possible to log a blank status.